src/aks-preview/azext_aks_preview/_params.py

```diff
@@ -370,11 +370,18 @@ def load_arguments(self, _):
             c.argument('message_of_the_day', type=str,
                        validator=validate_message_of_the_day)
 
-    for scope in ['aks nodepool show', 'aks nodepool delete', 'aks nodepool scale', 'aks nodepool upgrade', 'aks nodepool update']:
+    for scope in ['aks nodepool show', 'aks nodepool scale', 'aks nodepool upgrade', 'aks nodepool update']:
         with self.argument_context(scope) as c:
             c.argument('nodepool_name', type=str, options_list=[
                        '--name', '-n'], validator=validate_nodepool_name, help='The node pool name.')
 
+    with self.argument_context('aks nodepool delete') as c:
+        c.argument('nodepool_name', type=str, options_list=[
+            '--name', '-n'], validator=validate_nodepool_name, help='The node pool name.')
+        c.argument('ignore_pod_disruption_budget', options_list=[
+                   "--ignore-pod-disruption-budget", "-i"], action='store_true', is_preview=True,
+                   help='delete an AKS nodepool by ignoring PodDisruptionBudget setting')
```

Comment:
Consider using three state flag to support setting the value to None, False or True.